### DISCUSSION TITLE
Messenger and Slack?

### DISCUSSION THREAD

**Discussion by Lotsofjoy** [ferdium/ferdium-app#1394] [Q&A] [answered]
Hello, 

I'm a Franz refugee looking for a place to call home.

I need something that will work with 2 different FB messenger accounts simultaneously, Slack, and telegram. I can't tell from what I see on the site, if those are supported or not. Would love some input from people using it.

**Comment by vraravam** [comment, marked as answer]
Unless the website does some advanced things to dissuade the user from such activity, Ferdium itself doesn't impose such restrictions. Please see the home page of Ferdium for the message about configuring the same service twice or more - one for each login that you want to use
